classes/Meta.php

```diff
@@ -524,10 +518,26 @@ public static function completeMetaTags($metaTags, $defaultValue, Context $conte
         if (empty($metaTags['meta_title'])) {
             $metaTags['meta_title'] = $defaultValue;
         }
+        if (!empty($context->controller) && method_exists($context->controller, 'getTemplateVarPagination')) {
+            Meta::paginateMetaTitle($metaTags['meta_title']);
+        }
 
         return $metaTags;
     }
 
+    /**
+     * Add page number to meta_title
+     *
+     * @param string $meta_title
+     */
+    public static function paginateMetaTitle(&$meta_title)
```

Comment:
```suggestion
    private static function paginateMetaTitle(string $meta_title): string
```

Reply:
Because arguments passed by reference are perfect bug traps I suppose.
Do you confirm @Progi1984  ?